apps/coscrad-frontend/src/components/dynamicViews/commands/CommandExecutionPage.tsx

```diff
@@ -0,0 +1,12 @@
+import { ICommandFormAndLabels } from '@coscrad/api-interfaces';
+import { useLocation } from 'react-router-dom';
+import { ComamndForm } from './CommandForm/CommandForm';
+
+export const CommandExecutionPage = (): JSX.Element => {
+    const location = useLocation();
+
```

Comment:
Any use cases for useHistory() if you're navigating backwards from different views?